docs/lib/datastore/fragments/android/getting-started/configuration.md

```diff
@@ -2,12 +2,27 @@
 
 In your app's `onCreate` (or similar lifecycle callback), initialize Amplify:
 
+<amplify-block-switcher>
+<amplify-block name="Java">
+
 ```java
 Amplify.addPlugin(new AWSDataStorePlugin());
```

Comment:
Let's do the whole try/catch dance here as people will copy/paste.

Reply:
fixed